test/parallel/test-fs-chmod-mask.js

```diff
@@ -71,23 +71,23 @@ function test(mode, asString) {
   if (fs.lchmod) {
     const link = path.join(tmpdir.path, `lchmod-src-${suffix}`);
     const file = path.join(tmpdir.path, `lchmod-dest-${suffix}`);
-    fs.writeFileSync(file, 'test', 'utf-8');
+    fs.writeFileSync(file, 'test', { encoding: 'utf-8', mode: 0o444 });
     fs.symlinkSync(file, link);
 
     fs.lchmod(link, input, common.mustCall((err) => {
       assert.ifError(err);
-      assert.strictEqual(fs.lstatSync(link).mode & 0o777, mode);
+      assert.strictEqual(fs.lstatSync(link).mode & 0o444, 0o444);
     }));
   }
 
   if (fs.lchmodSync) {
     const link = path.join(tmpdir.path, `lchmodSync-src-${suffix}`);
     const file = path.join(tmpdir.path, `lchmodSync-dest-${suffix}`);
-    fs.writeFileSync(file, 'test', 'utf-8');
+    fs.writeFileSync(file, 'test', { encoding: 'utf-8', mode: 0o444 });
     fs.symlinkSync(file, link);
 
     fs.lchmodSync(link, input);
```

Comment:
Might also be nice to run the gamut, i.e.:
```js
    for (const m of [...Array(mode + 1).keys()]) {
      fs.lchmodSync(link, m);
      assert.strictEqual(fs.lstatSync(link).mode, m);
    }
```

Reply:
`fs.lstatSync(link).mode` returns `40960` and fails in the first iterations where m is `0`.

Reply:
Maybe:
```js
    for (const m of [...Array(mode).keys()]) {
      fs.lchmodSync(link, m + 1);
      assert.strictEqual(fs.lstatSync(link).mode & 0xfff, m);
    }
```
If this doesn't work, we can skip it.